vite-hardhat/hardhat.config.cts

```diff
@@ -158,10 +51,12 @@ const config: HardhatUserConfig = {
       accounts: vars.has('holesky') ? [vars.get('holesky')] : [],
     },
   },
+  noir: {
+    version: '0.36.0',
+  },
   paths: {
-    root: './',
-    sources: './artifacts',
-    artifacts: './artifacts/hardhat',
+    noir: 'circuit',
```

Comment:
if this is an official starter, i would stick to not overriding default path for noir. I would either keep it in noir folder or change the hardhat-plugin-noir default to "circuit". The thing is that hardhat-plugin-noir supports nargo workspaces, so "circuit" is not a good name. Maybe "circuits". But I like "noir" more

Reply:
I think I'll just make this a workspace, most of the projects I've seen end up adopting that structure

Reply:
Yeah, good idea. The plugin supports workspaces out of the box btw. It will event create a workspace Nargo.toml if you missed it